src/pages/cadastrar-demanda/cadastrar-demanda.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>cadastrar demanda</title>
+    <link rel="stylesheet" href="../../css/index.css">
+    <link rel="stylesheet" href="cadastrar-demanda.css">
+    <script type="module" src="../../components/header/header.js"></script>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+</head>
+
+<body>
+
+    <div class="container">
+        <div id="page-overlay"></div>
+        <header>
+            <header-component action="toggleOverlay"></header-component>
+        </header>
+
+        <main class="content">
+            <h1 class="title" id="cadastrar-demanda">cadastrar demanda</h1>
+
+            <form action="/demandas" method="post">
+
+                <div class="row">
+                    <div class="col-25">
+                        <label for="nome-demanda">Nome da demanda:</label>
+                    </div>
+                    <div class="col-75">
+                        <input type="text" id="nome-demanda" name="nome-demanda"  maxlength="30" required>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-25">
+                        <label for="tipo-demanda">Tipo de demanda:</label>
+                    </div>
+                    <div class="custom-radio">
+                        <input type="radio" id="presencial" name="tipo-demanda" value="presencial" required>
+                        <label for="presencial">Presencial</label>
+
+                        <input type="radio" id="remota" name="tipo-demanda" value="remota">
+                        <label for="remota">Remota</label>
+
+                        <input type="radio" id="hibrida" name="tipo-demanda" value="hibrida">
+                        <label for="hibrida">Híbrida</label>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-25">
+                        <label for="sobre-demanda">Descreva sua nescessidade:</label>
```

Comment:
Corrigir para `necessidade`
```suggestion
                        <label for="sobre-demanda">Descreva sua necessidade:</label>
```